index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header id="header" >
+        <span id="logo" >Robertson Arthur</span>
+        <button type="button" class="btn header-btn">
+            <img src="./icons/Icon-Menumenu-icon.png" alt="Menu-icon">
+        </button>
+    </header>
+
+    <main>
+        <section id="headline-section" >
+            <h1 class="headline-primary-text" >Hey There. I’m Robertson</h1>
+            <h4>I’am a Software Developer</h4>
+            <p>I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+
+            <div class="headline-social-media" >
+                <button type="button" >
+                    <img src="./icons/Disabledgithub-icon.png" alt="">
+                </button>
+                <button type="button">
+                    <img src="./icons/DisabledLinkedIn-icon.png" alt="">
+                </button>
+                <button type="button">
+                    <img src="./icons/Enabledtwo-fingers-icon.png" alt="">
+                </button>
+                <button type="button">
+                    <img src="./icons/Disabledtwitter-icon.png" alt="">
+                </button>
+                <button type="button">
+                    <img src="./icons/DisabledM-icon.png" alt="">
+                </button>
+            </div>
```

Comment:
In order to use semantic HTML tags, it is better if we make use of the unordered list, with list item tags inside.

- [ ] Kindly, replace the wrapper `div` tag with a `ul` tag and each individual link wrap it in a `li` tag;  

- [ ]  It will be more appropriate to use `a` tags for social media links and not `button` tags, as they really are links and not buttons in our project;